iphone/metadata/de-DE/release_notes.txt

```diff
@@ -1,41 +1,30 @@
+Search
+• Parking entrances are searchable
+• Improved street matching and results relevance
+• Search for grocery and houseware
 
-Suche
-• Die Suche nach Adressen in Tschechien und der Slowakei wurde korrigiert
-• Verbesserter Straßenabgleich
-• Touristeninformation, Kultstätte, Park, Informationstafel und Touristenkarte wurden aus der Kategorie Sehenswürdigkeiten entfernt
-• Doppelte Suchergebnisse werden nicht mehr gefiltert, außer bei sehr nahen Haltestellen
-• Besseres Ranking bei der Suche nach Kategorien
-• Behobene Suche nach einem Hausnamen
+Map styling:
+• New or updated icons for factory, winery, bookcase, library, bicycle repair station, bicycle rental, bicycle parking, sauna, lottery, bookmaker, massage, sewing, books, underground parking, disabled parking space, gate, excrement bags vending
+• Added vehicle inspection, public bath, cycle barrier, waterways (drain, ditch, moat, wastewater), aquarium, assembly point, amusement arcade, Mormon churches, visitor center, industrial mine, historic objects (cannon, anchor, stone, aircraft, tank, mine, wreck, locomotive)
+• Added bridges and tunnels rendering for cycleways, footways, paths, tracks, bridleways, steps
+• Render barrier=ditch differently from waterway=ditch
+• Reduced priorities of river names (and of other lower-importance lines) in navigation mode
+• Tuned road names and road shield priorities
+• Fixed icons display for parking buildings
+• Reduced visibility of minor and private parking icons, removed area fill for underground parking
+• Reduced fire hydrants visibility
+• Don’t display house numbers for buildings that are too small for the current zoom level
+• Don't display star icon for squares
+• Fine-tune house numbers font and drawing priority
+• Added highway-pedestrian, bridleway, and cycleway lines for detailed zooms (z18-) in navigation mode
+• Removed highway-path/track/footway/steps tunnels rendering in navigation mode
 
-Stile
-• Fährenrouten zur Weltkarte hinzugefügt
-• Der quadratische Hintergrund von Parksymbolen wurde entfernt
-• Separate Icons für kostenpflichtige und private Parkplätze hinzugefügt
-• Das Fahrradparkplatz-Symbol unterscheidet sich nun deutlicher vom Motorrad-Symbol
-• Das Symbol für den Parkeingang wurde aktualisiert
-• Für Parkplätze am Straßenrand und auf Fahrspuren wird nun ein kleineres Symbol verwendet, das beim Heranzoomen später erscheint
-• Ein Motorradparkplatz• und Parkeingangssymbol wurde zum Navigationsmodus hinzugefügt
-• Die Sichtbarkeit der Autobahn-Links im Navigationsmodus wurde verbessert
-• Verbesserte Farben für Autobahn
-• Zu kleine Isolinien bei Übersichts-Zoomstufen wurden entfernt
-• Teheraner U-Bahn-Symbol hinzugefügt
-• Überarbeitetes Prioritätensystem, insbesondere für POIs
-• Viele weitere Korrekturen und Verbesserungen
-
-Verfolgen Sie unsere Neuigkeiten unter https://omaps.org/de/news
+…and more smaller fixes and improvements, check https://omaps.org/news for details!
```

Comment:
... und weitere kleine Fehlerbehebungen und Verbesserungen, mehr Infos unter https://omaps.org/news

Reply:
Forget this last one, I marked the wrong section here....